Formula/webdis.rb

```diff
@@ -0,0 +1,37 @@
+class Webdis < Formula
+  desc "Redis HTTP interface with JSON output"
+  homepage "http://webd.is/"
+  url "https://github.com/nicolasff/webdis/archive/0.1.2.tar.gz"
+  sha256 "8e46093af006e35354f6b3d58a70e3825cd0c074893be318f1858eddbe1cda86"
+
+  depends_on "libevent"
+
+  def install
+    system "make"
+    bin.install "webdis"
+
+    inreplace "webdis.prod.json" do |s|
+      s.gsub! "/var/log/webdis.log", "#{var}/log/webdis.log"
```

Comment:
@freeatnet I merged a documentation change about this (https://github.com/Homebrew/brew/issues/131) and dug up the relevant documentation justifying the way we choose to run this project. I've been a maintainer on this project for just under 7 years and this has been one of the top 10 most unpleasant interactions I've had on Homebrew while being a Homebrew maintainer. While I appreciate there are questions you may like to have answered I'd rather we all just moved on so am locking this thread. I hope you'll contribute to Homebrew again in future but I understand if you don't. I'd also appreciate if you could avoid the unkind words towards specific maintainers or the project on your Twitter. Thanks again for your contribution to Homebrew and I hope you have a nice weekend 🙇